includes/admin/discounts/add-discount.php

```diff
@@ -210,6 +210,9 @@
 						<span class="description"><?php esc_html_e( 'Prevent customers from using this discount more than once.', 'easy-digital-downloads' ); ?></span>
 					</td>
 				</tr>
+				
+				<?php do_action( 'edd_add_discount_form_after_use_once' ); ?>
```

Comment:
I know the others in this file don't have them, but can you please add a docblock with `@since 3.0` ?